crates/protocol/protocol/src/fee.rs

```diff
@@ -77,7 +77,7 @@ pub fn calculate_tx_l1_cost_bedrock_empty_scalars(
     base_fee: U256,
     l1_fee_scalar: U256,
 ) -> U256 {
-    if input.is_empty() || input.first() == Some(&0x7F) {
+    if input.is_empty() || input.first() == Some(&0x7E) {
```

Comment:
this exists as a constant here

https://github.com/alloy-rs/op-alloy/blob/88c6c2efc063a7baf14b5a84829157dd9a5d1d75/crates/consensus/src/transaction/tx_type.rs#L10-L10

useful to avoid these things :)